README.md

```diff
@@ -158,6 +166,80 @@ The emitter has to be provided when creating an instance of `XRayRecorder`:
 let recorder = XRayRecorder(emitter: XRayNoOpEmitter())
 ```
 
+### Context propagation
+
+Unlike in other X-Ray SDKs, `XRayRecorder` in AWS X-Ray SDK for Swift does not expose (thread local) current `Segment`. 
```

Comment:
👍 good, also because thread locals would not work well in swift (with NIO / Queues / some form of upcoming "Swift Concurrency")